tiledb/sm/filesystem/vfs.cc

```diff
@@ -1593,6 +1615,7 @@ Status VFS::set_multipart_upload_state(
 }
 
 Status VFS::flush_multipart_file_buffer(const URI& uri) {
+  auto op = start_operation(uri, "flush_multipart_file_buffer");
```

Comment:
Note that when we change the name of start_operation, we'll have to rename op as well. This object is not the operation.

Reply:
Done.